addons/website/static/src/systray_items/edit.js

```diff
@@ -0,0 +1,114 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+
+const { Component, onMounted, useState, useEffect } = owl;
+
+export class EditSystray extends Component {
+    setup() {
+        this.websiteService = useService('website');
+        this.actionService = useService('action');
+        this.websiteContext = useState(this.websiteService.context);
+
+        this.state = useState({
+            isLoading: false,
+        });
+
+        useEffect((edition) => {
+            if (edition) {
+                this.state.isLoading = true;
+            }
+        }, () => [this.websiteContext.edition]);
+
+        useEffect((snippetsLoaded) => {
+            if (snippetsLoaded) {
+                this.state.isLoading = false;
+            }
+        }, () => [this.websiteContext.snippetsLoaded]);
+
+        onMounted(() => {
+            this.websiteService.editedObjectPath = null;
+        });
+    }
+
+    get translatable() {
+        return this.websiteService.currentWebsite &&
+            this.websiteService.currentWebsite.metadata.translatable;
+    }
+
+    get label() {
+        if (this.translatable) {
+            return this.env._t('Edit master');
+        }
+        return this.isDropdown() ? this.env._t('Edit page') : this.env._t('Edit');
+    }
+
+    editInBackend() {
+        const { metadata: { mainObject } } = this.websiteService.currentWebsite;
+        this.websiteService.editedObjectPath = this.websiteService.contentWindow.location.pathname;
+        this.actionService.doAction({
+            res_model: mainObject.model,
+            res_id: mainObject.id,
+            views: [[false, "form"]],
+            type: "ir.actions.act_window",
+            view_mode: "form",
+        });
+    }
+
+    getEditActionName() {
+        // Fallback if the function is not overridden.
+        return this.env._t('Edit in backend');
```

Comment:
We don't want to reintroduce this term "In backend" when we are already in backend and ou clients don't know what is a "backend"